### DISCUSSION TITLE
Simplify GraphQL Credential Field

### DISCUSSION THREAD

**Discussion by OwenMelbz** [craftcms/cms#8931] [Ideas]
My personal opinion is that the GraphQL credential field might be over engineered currently and can be simplified... maybe.

Currently it looks like

![image](https://user-images.githubusercontent.com/1094740/94683441-744c6d00-031e-11eb-9deb-d8ffa41b1a15.png)

When you try to select the Token to use in other systems, or .env files etc It forces. you to select the whole thing, which is annoying when you're trying to select the text you want. It breaks the default expected behavior of the browser.

It *Might* also have a11y consequences...?

Additionally when you click "Copy" it copies the whole string also leaving you in a situation like...

![image](https://user-images.githubusercontent.com/1094740/94683606-b5448180-031e-11eb-8ea5-40099b562b60.png)

Basically there's no way to get the Token out without first copying it, then pasting it somewhere, then manually deleting the extra text.

Which I imagine *most* use cases need the Token and not a raw header as I'd have thought most people would be using HTTP clients which allows the headers to be set in a non-raw fashion.

It might be that the "Copy" is left as it is and copies the whole string, but I think removing the "forcing select everything" could be a nice middle ground?

Thanks
